### ISSUE TITLE
Fails to mount non existing file

### ISSUE THREAD

**Issue by hotio** [yuichielectric/dive-action#3]
When not using a `config-file`, the docker run command tries to mount a default file regardless, which fails.

**Comment by hotio** [comment]
Or am I misinterpreting the usage, maybe a config file is always required, but I'm not required to configure one?

**Comment by hotio** [comment]
It seems that Dive always expects a config file...maybe provide a default config file if the user does not make use of `config-file` input.....?

**Comment by yuichielectric** [comment]
@hotio Sorry for the delay in replying, the config-file isn't required, dive has a default CI setting, so it should be run using that setting.
I've made some modifications since then, could you try again with the following settings? 

I ran it in this repository without config-file and made sure it worked.
https://github.com/yuichielectric/dockerfile-ci-sample/blob/cb6d6a5f7500187b175519f3aa61ee9c866de810/.github/workflows/dive-ci.yml

**Comment by hotio** [comment]
Still getting the same error with:
```

      - name: Dive Analysis
        uses: yuichielectric/dive-action@0.0.3
        with:
          image: "${{ steps.prep.outputs.tag }}"

```
https://github.com/hotio/docker-hd-idle/runs/1175314154?check_suite_focus=true#step:10:23

**Comment by yuichielectric** [comment]
Thanks, I was able to reproduce it in my repository. I'll fix it soon.

**Comment by yuichielectric** [comment]
I've fixed this bug and released 0.0.4. I confirmed that it works fine without passing `config-file` parameter. Can you try again with `0.0.4` ?

**Comment by hotio** [comment]
yep works now
